oap-server/server-starter/src/main/resources/ui-initialized-templates/menu.yaml

```diff
@@ -47,6 +47,16 @@ menus:
         description: Observe the virtual message queue servers which are conjectured by language agents through various plugins.
         documentLink: https://skywalking.apache.org/docs/main/next/en/setup/service-agent/virtual-mq/
         i18nKey: general_service_virtual_mq
+  - title: Workflow Scheduler
+    icon: workflow_scheduler
+    description: Provide monitoring for workflow scheduling systems
+    i18nKey: workflow_scheduler
+    menus:
+      - title: Airflow
+        layer: AIRFLOW
+        description: Provide Airflow monitoring through OpenTelemetry otlp Receiver.
+        documentLink: https://skywalking.apache.org/docs/main/next/en/setup/backend/backend-airflow-monitoring/
+        i18nKey: workflow_scheduler_airflow
```

Comment:
Same here, you should not randomly add it as the first one. It should align with the one I suggested for doc menu.